src/renderer/shims/LocalShim.ts

```diff
@@ -81,12 +81,24 @@ export class LocalShim implements SimulatorInterface {
     public Aircraft = {}
 
     public GetStoredData = (key: string): any => {
-        if (key === 'A32NX_CONFIG_SELF_TEST_TIME') {
-            return 2;
+        // if (key === 'A32NX_CONFIG_SELF_TEST_TIME') {
+        //    return 2;
+        //}
+
+        try {
+            const item = window.localStorage.getItem(key);
+            return item ? JSON.parse(item) : null;
+        } catch (error) {
+            console.log(error);
+            return 0;
         }
-        return 0;
     }
 
-    public SetStoredData = () => {
+    public SetStoredData = (key: string, value: any): void => {
+        try {
+            window.localStorage.setItem(key, JSON.stringify(value));
+        } catch (error) {
+            console.log(error);
+        }
```

Comment:
Should return `null`

Reply:
Has to be aligned with `NXDataStore,get(key, defaultValue)` to correctly return the default value in case key is not set yet:
```
    static get(key, defaultVal) {
        const val = GetStoredData(`A32NX_${key}`);
        if (!val) {
            return defaultVal;
        }
        return val;
    }
```
Returning an empty string works the same as `null` but I can change it to an empty string.

Reply:
It should match the MSFS code it is substituted for as far as I'm concerned, because that is how it works in the sim. If there is other code that doesn't work with this you have found a bug in the other code.